tools/generators/legacy/src/Generator/AddTargets.swift

```diff
@@ -192,27 +192,17 @@ Product for target "\(key)" not found in `products`
             return nil
         }
 
-        let copyOutputs: String
-        if let bazelBasename = outputs.productBasename {
-            copyOutputs = #"""
-else
-  "$BAZEL_INTEGRATION_DIR/copy_outputs.sh" \
-    "\#(Generator.bazelForcedSwiftCompilePath)" \
-    "\#(bazelBasename)" \
-    "\#(productType.rsyncExcludeFile)"
-"""#
-        } else {
-            copyOutputs = ""
-        }
-
         let shellScript = #"""
 set -euo pipefail
 
 if [[ "$ACTION" == "indexbuild" ]]; then
   cd "$SRCROOT"
 
   "$BAZEL_INTEGRATION_DIR/generate_index_build_bazel_dependencies.sh"
-\#(copyOutputs)
```

Comment:
Because of this change, you'll still need to run `./test/update_all_fixtures.sh` again.